lib/Helpers.js

```diff
@@ -114,9 +114,9 @@ class Helpers {
         break
       }
 
-      for (let i = 0; i < maxRetries; i++) {
+      for (let i = 0; i <= maxRetries; i++) {
         response = await this[kClient].scroll({
-          ...querystring,
+          ...this.filterQueryStringForScrolling(querystring),
```

Comment:
I would configure the parameters in place, without calling an external function :)
```suggestion
          scroll: querystring.scroll,
          rest_total_hits_as_int: querystring.rest_total_hits_as_int || querystring.restTotalHitsAsInt
```

Reply:
Sure can, I haven't downloaded the project, yet. This fixes were quick changes made through github's online editor 😊
I'll do the proper changes and commit to this PR.